pages/event-types/[type].tsx

```diff
@@ -546,6 +546,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                         }
                         id="length"
                         required
+                        min="1"
```

Comment:
1 is probably not a great default, a 1 minute meeting is a bit unrealistic. The smallest meeting time could be 5 - 15m?

Reply:
```suggestion
                        min="5"
```

Reply:
@PeerRich suggested 10 minutes would be good a good min value for the duration.